tests/integration.rs

```diff
@@ -410,6 +410,46 @@ mod tests {
         )
     }
 
+    fn prepare_vhost_user_net_daemon(
+        tmp_dir: &TempDir,
+        ip: &str,
+        tap: Option<&str>,
+        num_queues: usize,
+    ) -> (std::process::Child, String) {
+        let vunet_socket_path = String::from(tmp_dir.path().join("vunet.sock").to_str().unwrap());
+
+        // Start the daemon
+        let child = if let Some(tap_str) = tap {
+            Command::new("target/release/cloud-hypervisor")
+                .args(&[
+                    "--net-backend",
+                    format!(
+                        "tap={},ip={},mask=255.255.255.0,sock={},num_queues={},queue_size=1024",
+                        tap_str, ip, vunet_socket_path, num_queues
+                    )
+                    .as_str(),
+                ])
+                .spawn()
+                .unwrap()
+        } else {
+            Command::new("target/release/cloud-hypervisor")
+                .args(&[
+                    "--net-backend",
+                    format!(
+                        "ip={},mask=255.255.255.0,sock={},num_queues={},queue_size=1024",
+                        ip, vunet_socket_path, num_queues
+                    )
+                    .as_str(),
+                ])
+                .spawn()
+                .unwrap()
+        };
```

Comment:
Just a nit but you could factorize a little bit more like that:
```suggestion
        let net_params = if let Some(tap_str) = tap {
            format!(
                "tap={},ip={},mask=255.255.255.0,sock={},num_queues={},queue_size=1024",
                tap_str, ip, vunet_socket_path, num_queues
            )
        } else {
            format!(
                "ip={},mask=255.255.255.0,sock={},num_queues={},queue_size=1024",
                ip, vunet_socket_path, num_queues
            )
        }
        let child = Command::new("target/release/cloud-hypervisor")
            .args(&[
                "--net-backend",
                net_params.as_str(),
            ])
            .spawn()
            .unwrap();
```